server/dbutil/dbutil.go

```diff
@@ -49,14 +79,54 @@ const insertFilePairs = `INSERT INTO file_pairs (
 		score, diff, experiment_id ) VALUES
 		($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15)`
 
+// Open returns a DB from the connection string.
+// With checkExisting it will fail if the DB does not exist
+func Open(connection string, checkExisting bool) (DB, error) {
+	sqliteReg := regexp.MustCompile(`^sqlite://(.+)$`)
+	psReg := regexp.MustCompile(`^postgres(ql)?:.+$`)
```

Comment:
you are checking only for prefix, right?
Would it be easier to use `string.HasPrefix`?
and for `sqlite` to remove prefix you can use `string.TrimPrefix`

just a suggestion, up to you.

Reply:
I like to use regexp if there is any excuse :D